lib/gamesDbCalls.js

```diff
@@ -1,34 +1,39 @@
 const db = require("../models");
+const shuffle = require('lodash/shuffle');
+const gameSeed = () => require('../engine/seed');
 
 // used in controllers/gamesController.js
 module.exports = {
   // lists all current matching games: for joining/watching games
-  findAllPromise: (query) => {
+  findAll: (query) => {
     return db.Game.find(query);
-  }
+  },
   // get game matching your selected game id
   // returned as part of auth cookie info for each user
-  findByIdPromise: (id) => {
-    return db.Game.findById({id});
-  }
+  findById: (id) => {
+    return db.Game.findById(id);
+  },
   // make a new game
   // if this doesn't work as expected, this might be the problem:
   // https://github.com/Automattic/mongoose/issues/5022
-  createPromise: (newState) => {
-    return db.Game.create(newState);
-  }
+  create: () => {
+    let newGame = gameSeed();
+    newGame.cards.deck = shuffle(shuffle(newGame.cards.deck)); // Double shuffle em
+    newGame.cards.excluded = newGame.cards.deck.shift();
+    return db.Game.create(newGame);
+  },
   // run this to play the game every turn or socket.io update cycle
   // example usage with express:
   // function(req, res) {
   //   updatePromise(`123foobar`, {user: {}, }).then((data) => res.json(data));
   // }
-  updatePromise: (id, newState) => {
-    return db.Game.findOneAndUpdate({id}, newState);
-  }
+  update: (_id, newState) => {
```

Comment:
Is there a reason to keep the underscore with the `id` param here?

Reply:
I believe in Mongoose, the id that is automatically generated for documents is called `_id`, so this just makes it so that you can use the object deconstruction shortcuts.

Reply:
That's clever, I hadn't thought to try doing that!